projects/canopy-test-app/src/app/app.component.html

```diff
@@ -250,6 +250,53 @@
             </lg-card-header>
           </lg-card>
 
+          <div lgContainer>
+            <div lgRow>
+              <div lgCol="12" lgColLg="8" lgColLgOffset="2" lgColMd="10" lgColMdOffset="1">
+                <lg-card lgPadding="none">
```

Comment:
Part of me is wondering if there's a precarious balance of margin/padding directives going on here, and it relies on the developer getting it just right, or ensuring that they copy and paste this example exactly. I'm not sure if you discussed this on the call and apologies if you have, but I do wonder it it's worth considering having a card variant for the Form Journey, which sets the classes on itself and it's children to make sure it displays with all the correct margin and padding. 

It could even apply the border on the header/footer, in case there are scenarios where we don't want a border (e.g. `--with-border`), although that could be [YAGNI](https://www.wikiwand.com/en/You_aren%27t_gonna_need_it). 

What do you think? Again apologies if I'm treating old ground at this late stage!

Reply:
For me the form journey variant still felt like one step too far into a specific use case, I may be wrong on this as @alexcanning  has pointed out it will be a common pattern. However I wouldn't want us to build up a catalogue of variants that we have to support.

We still haven't really looked at the card design holistically, it could be that they should all have the same paddings and margins but we just don't know it yet. Might be worth a real quick check in on that point only and see what effect it would have on the other card examples.

Reply:
Not sure what's the best course of action here. Without the variant, making use of canopy Grid, we can get a variety of cards out of the box, on the other hand the variant seems to be more clean or maybe it looks like that to me. I am good to go with both approaches, both of them giving great flexibility to the user. Not sure what you mean by "quick check in on that point only and see what effect it would have on the other card examples." as this is how I've built it once, with a variant, and I did not modify the other card examples.
I'm good with either implementation so waiting for thoughts on this one. :)

Reply:
I'm a bit stuck too, What I was wondering is..

1. We make these the default spacings for the card-header: `<lg-card-header lgPadding="sm" lgPaddingBottom="xs" lgMarginBottom="lg">`(I moved the marginTop from the lgContainer)
2. If the other storybook examples weren't badly affected then we proceed with that. The other examples with card headers have never had any formal design review so I'm not too worried about visual changes to them.

I think one we have a sensible margin and padding setup that we keep consistent that should make the cards a bit easier. Then we could have a form journey variant, but I wonder if it would actually need to be a component in it's own right as it will probably need all the grid markup to be responsive, we could defer that one for now to ease this in. Another opinion would be handy I think on the form journey question.

Reply:
After a short discussion with Andy, I moved all the added spacing via margins/paddings directives just to the <lg-card-header> and the other cards did not suffer any changes from a design point of view.
Regarding the variant, we decided not to have one at the moment, but can be revisited in the future if and when we get further designs on the other cards like: standard, product and so on.

Reply:
Sounds good to me 🙂